pkg/addons/applier.go

```diff
@@ -50,11 +52,12 @@ type applier struct {
 
 // TemplateData is data available in the addons render template
 type templateData struct {
-	Config         *kubeoneapi.KubeOneCluster
-	Certificates   map[string]string
-	Credentials    map[string]string
-	InternalImages *internalImages
-	Resources      map[string]string
+	Config             *kubeoneapi.KubeOneCluster
+	Certificates       map[string]string
+	Credentials        map[string]string
+	CredentialsEnvVars string
```

Comment:
```suggestion
	MachineControllerCredentialsEnvVars string
```

I know it's quite long name, but it makes the purpose much more transparent.

Reply:
@kron4eg Changed, PTAL.